scenes/tileset.tres

```diff
@@ -29,10 +29,13 @@ tile_data = PackedInt32Array(0, 327681, 3, 1, 393217, 3, 2, 458753, 3)
 [sub_resource type="TileSetAtlasSource" id="TileSetAtlasSource_sugp2"]
 texture = ExtResource("1_glnwb")
 texture_region_size = Vector2i(64, 64)
-0:0/next_alternative_id = 2
+0:0/next_alternative_id = 3
 0:0/0 = 0
 0:0/0/z_index = -1
 0:0/0/physics_layer_2/polygon_0/points = PackedVector2Array(-32, -32, 32, -32, 32, 32, -32, 32)
+0:0/2 = 2
+0:0/2/z_index = -1
+0:0/2/physics_layer_0/polygon_0/points = PackedVector2Array(-32, 3.85675, 32, 4.40771, 32, 32, -32, 32)
```

Comment:
@JuanFdS Following your change in commit d4c8aa459fe865812e4d3ee287c64eca5681563d, should this half-collidable water tile also use physics layer 2, in case it gets used in a level with ink blobs?